float32 servo0
float32 servo1
float32 servo2
float32 servo3
float32 servo4
float32 servo5